internals/api/service.go

```diff
@@ -24,13 +24,22 @@ var (
 	ErrAccessDeniedToKMSKey = errAPI.Code("access_denied").StatusError("access to KMS key is denied", http.StatusForbidden)
 )
 
+type ServiceType string
+
+// Service types
+const (
+	ServiceTypeAWS ServiceType = "aws"
+	ServiceTypeRSA ServiceType = "rsa"
```

Comment:
@mackenbach and I just discussed that it makes more sense to expose this to the user as type `key`. Then the user is not bothered with _what_ type of key. The actual key type used could be stored in the metadata, if we wish.